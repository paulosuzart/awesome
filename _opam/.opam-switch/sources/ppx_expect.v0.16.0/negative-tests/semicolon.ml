let%expect_test _ =
  print_string "one";
  [%expect {| two |}]
;;
