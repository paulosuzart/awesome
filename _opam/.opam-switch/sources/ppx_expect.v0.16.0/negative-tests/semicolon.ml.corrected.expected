let%expect_test _ =
  print_string "one";
  [%expect {| one |}]
;;
